invenio_rdm_records/oaiserver/services/permissions.py

```diff
@@ -6,16 +6,16 @@
 # it under the terms of the MIT License; see LICENSE file for more details.
 
 """Permissions for OAI-PMH service."""
-
+from invenio_administration.generators import Administration
 from invenio_records_permissions import BasePermissionPolicy
 from invenio_records_permissions.generators import Admin, AnyUser, SystemProcess
 
 
 class OAIPMHServerPermissionPolicy(BasePermissionPolicy):
     """OAI-PMH permission policy."""
 
-    can_read = [Admin(), SystemProcess()]
-    can_create = [Admin(), SystemProcess()]
-    can_delete = [Admin(), SystemProcess()]
-    can_update = [Admin(), SystemProcess()]
-    can_read_format = [Admin(), SystemProcess()]
+    can_read = [Admin(), Administration(), SystemProcess()]
```

Comment:
@kpsherva this means that the permission to access backoffice needs to be explicitily added everywhere... Can you double-check if it was documented?